argyrogounari/assignment1/Question1.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Scanner;
+import java.util.*;
+
+public class Question1 {
+
+	public static void main(String[] args) {
+		
+		Scanner reader = new Scanner(System.in);  
+		System.out.println("Enter your strings: ");
+		String sentence1 = reader.nextLine();
+		String sentence2 = reader.nextLine();
+		
+		reader.close(); 
+		
+		if (anagram(sentence1,sentence2)) {
+			System.out.printf("'%s' and '%s' are anagrams",sentence1, sentence2);
+		} else {
+			System.out.printf("'%s' and '%s' are NOT anagrams",sentence1, sentence2);
+		}
+
+	}
+	
+	public static boolean anagram(String sentence1, String sentence2){
+		
+		ArrayList<String> sentence1Words = new ArrayList<String>(Arrays.asList(sentence1.split(" ")));
+		ArrayList<String> sentence2Words = new ArrayList<String>(Arrays.asList(sentence2.split(" ")));
+		
+		int anagram = 0;
+        for (int i = 0; i < sentence1Words.size(); i++){
```

Comment:
Try to make sure that the indentation lines up, otherwise it can be very difficult for a reader to parse code quickly.